docs/source/deployment/airflow.md

```diff
@@ -0,0 +1,181 @@
+
+```Dockerfile
+ENV KEDRO_LOGGING_CONFIG="conf/logging.yml"
+
+RUN pip install --user dist/new_kedro_project-0.1-py3-none-any.whl
+```
+
+4. Navigate to `kedro-airflow-spaceflights` folder and launch the local Airflow cluster with Astronomer
+
+```shell
+cd kedro-airflow-spaceflights
+astro dev start
+```
+
+5. Visit the Airflow Webserver UI at its default address, http://localhost:8080, using the default login credentials: username and password both set to `admin`. There, you'll find a list of all DAGs. Navigate to the `new-kedro-project` DAG, and then press the `Trigger DAG` play button to initiate it. You can then observe the steps of your project as they run successfully:
+
+![](../meta/images/kedro_airflow_dag_run.png)
+
+6. The Kedro project was run inside an Airflow Docker container, and the results are stored there as well. To copy these results to your host, first identify the relevant Docker containers by listing them:
+```shell
+docker ps
+```
+Select the container acting as the scheduler and note its ID. Then, use the following command to copy the results, substituting `d36ef786892a` with the actual container ID:
+
+```shell
+docker cp  d36ef786892a:/usr/local/airflow/data/ ./data/
+```
+
+
+7. To stop the Astro Airflow environment, you can use the command:
+```shell
+astro dev stop
+```
+
+### Deployment to Astro Cloud
+
+You can easily deploy and run your project on Astro Cloud, the cloud infrastructure provided by Astronomer, by following these steps:
+
+1. Log in to your account on the [Astronomer portal](https://www.astronomer.io/) and create a new deployment if you don't already have one:
+![](../meta/images/astronomer_new_deployment.png)
+
+2. Use the Astro CLI to log in to your Astro Cloud account:
+```shell
+astro auth
+```
+You will be redirected to enter your login credentials in your browser. Successful login indicates that your terminal is now linked with your Astro Cloud account:
+
+![](../meta/images/astronomer_connected.png)
+
+3. To deploy your local project to the cloud, navigate to the `kedro-airflow-spaceflights` folder and run:
+```shell
+astro deploy
+```
+
+4. At the end of the deployment process, a link will be provided. Use this link to manage and monitor your project in the cloud:
+
+![](../meta/images/astronomer_cloud_deployment.png)
+
+## How to run a Kedro pipeline on Apache Airflow using a Kubernetes cluster
+
+The `kedro-airflow-k8s` plugin from GetInData | Part of Xebia enables you to run a Kedro pipeline on Airflow with a Kubernetes cluster. The plugin can be used together with `kedro-docker` to prepare a docker image for pipeline execution. At present, the plugin is available for versions of Kedro < 0.18 only.
```

Comment:
cc @marrrcin @lasica

Reply:
Yeah, I think that GiD was going to deprecate this plugin as the same could be already achieved (not as easily unfortunately) with the official plugin. @Lasica

Reply:
Let's continue the conversation in https://github.com/kedro-org/kedro-plugins/issues/652 then, to see to what extent the official plugin can bridge the gap.